src/locales/sk.js

```diff
@@ -344,7 +344,7 @@ export default {
     "saintVitusMartyr": "Svätého Víta, mučeníka",  // austria, czechRepublic, germany, switzerland
```

Comment:
BTW, you wanted to remove my comments, don’t you? :smiley:

Reply:
yes, but it's not urgent as long as there is no work in progress to implement i18n.

Reply:
I’ll leave it up to you. :smiley: